mypyc/lib-rt/exc_ops.c

```diff
@@ -189,38 +189,6 @@ void CPy_TypeError(const char *expected, PyObject *value) {
     }
 }
 
-// These functions are basically exactly PyCode_NewEmpty and
-// _PyTraceback_Add which are available in all the versions we support.
```

Comment:
This comment still applies to the next function I believe. Not sure what optimization this is talking about.